func/python/hash_check.py

```diff
@@ -0,0 +1,14 @@
+import hashlib
```

Comment:
Hashlib is another internal CPython library that lives in `lib-dynload`, hence I needed to check whether it was _just_ `ctypes`, or all `lib-dynload` libraries.